packages/trader/src/App/Containers/trade-settings-extensions.jsx

```diff
@@ -19,19 +19,19 @@ const ChartSettingContainer = Loadable({
 //     loading: UILoader,
 // });
 
-class TradeSettingsExtensions extends React.Component {
-    populateSettings = () => {
-        const { populateSettingsExtensions } = this.props;
+const TradeSettingsExtensions = ({ populateSettingsExtensions, store }) => {
+    const populateSettings = () => {
+        const value = ({ ...props }) => (
+            <MobxContentProvider store={store}>
+                <ChartSettingContainer {...props} />
+            </MobxContentProvider>
+        );
```

Comment:
just a suggestion
it's better if you define this const outside the TradeSettingsExtensions